.changeset/sixty-bananas-sit.md

```diff
@@ -0,0 +1,6 @@
+---
+"@rocket.chat/meteor": patch
+"@rocket.chat/model-typings": patch
```

Comment:
```suggestion
"@rocket.chat/meteor": minor
"@rocket.chat/model-typings": minor
```
Since this is a new feature (and not a fix), we should mark this as minor changes instead